doc/psbt.md

```diff
@@ -82,9 +82,10 @@ hardware implementations will typically implement multiple roles simultaneously.
   transactions.
 - **`decodepsbt`** is a diagnostic utility RPC which will show all information in
   a PSBT in human-readable form, as well as compute its eventual fee if known.
-- **`analyzepsbt`** is a utility RPC that examines an RPC and reports the
-  next steps in the workflow if known, computes the fee of the resulting
-  transaction, and estimates the weight and feerate if possible.
+- **`analyzepsbt`** is a utility RPC that examines a PSBT and reports the
```

Comment:
shouldn't it be "an utility RPC" ?

Reply:
Seems not: https://www.englishforums.com/English/AUtilityVsAnUtility/hrmlw/post.htm

Reply:
ok
i'm only more confused after reading that :laughing:  in any case, it's not a blocker for me

Reply:
I thought it might be a British English vs American English thing, but turns out the Oxford site here https://en.oxforddictionaries.com/definition/utility says:

```md
3 Computing
A utility program.
```